package.json

```diff
@@ -42,6 +45,7 @@
   ],
   "dependencies": {
     "@carbon/colors": "^10.37.1",
+    "@carbon/ibmdotcom-web-components": "^1.22.0-nightly",
```

Comment:
I use the nightly because this is not released yet:
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/9089

Reply:
Updated!